user_manual/groupware/mail.rst

```diff
@@ -17,7 +17,7 @@ Switch layout
 1. Visit mail settings
 2. Choose between *List*, *Vertical split* and *Horizontal split*
 
-.. figure:: images/mail_layout.png
+.. figure:: ../images/mail_layout.png
```

Comment:
The file is located at https://github.com/nextcloud/documentation/blob/e20930354174492c20c95dcb8ee621df3728e6a2/user_manual/groupware/images/mail_layout.png, not https://github.com/nextcloud/documentation/blob/e20930354174492c20c95dcb8ee621df3728e6a2/user_manual/images/mail_layout.png